tools/webdriver/webdriver/bidi/modules/script.py

```diff
@@ -78,11 +78,11 @@ def __init__(self, context: str, sandbox: Optional[str] = None):
 class SerializationOptions(Dict[str, Any]):
     def __init__(
             self,
-            max_dom_depth: Optional[int] = None,
+            max_dom_depth: Optional[Union[int, None]] = 0,
             max_object_depth: Optional[int] = None,
```

Comment:
This line and the next need an update as well. Or it's not clear to me based on the description of the PR why you only modified the above one.